common/dynamicconfig/constants.go

```diff
@@ -1197,6 +1197,12 @@ Set to zero to disable proactive unload.`,
 		0.0,
 		`MatchingQueryWorkflowTaskTimeoutLogRate defines the sampling rate for logs when a query workflow task times out. Since
 these log lines can be noisy, we want to be able to turn on and sample selectively for each affected namespace.`,
+	)
+	PhysicalTaskQueueInfoByBuildIdTTL = NewTaskQueueDurationSetting(
```

Comment:
I don't think we should use "Physical" word in here, physical queue is the most granular level of TQ and it does not span over multiple partitions. What we are caching here is the aggregated info across partitions, but for a single build. I think we should call it just `TaskQueueInfoCacheTTL` and the fact is per build or not is not that relevant here.

Reply:
I support this (even though I already asked to change the name lol, but I agree that "physical" should not be what users think about, and dynamic config is one of the user-facing areas for open source users and for the internal teams)